src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+
 public class SalaryInfo {
+    private static final String DATA_RECORD_SPLIT_REGEX = " ";
+    private static final int SALARY_OFFSET = 0;
+    private static final int DATE_COLUMN = 0;
+    private static final int NAME_COLUMN = 1;
+    private static final int WORKING_HOURS_COLUMN = 2;
+    private static final int SALARY_RATE_COLUMN = 3;
+    private int[] totalSalary = new int[] {};
+    private String dataPattern = "dd.MM.yyyy";
+    private DateTimeFormatter simpleDateFormat = DateTimeFormatter.ofPattern(dataPattern);
```

Comment:
```suggestion
    private DateTimeFormatter simpleDateFormat = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
or make string constant